src/main/java/com/github/onsdigital/KeyLocatorImpl.java

```diff
@@ -60,12 +59,11 @@ public SigningKeyResolverImpl(Map<String, String> signingKeys) {
      * header and claims.
      *
      * @param jwsHeader the header of the JWS to validate
-     * @param claims    the claims (body) of the JWS to validate
      * @return the signing key that should be used to validate a digital signature for the Claims JWS with the specified
      * header and claims.
```

Comment:
I could be reading this incorrectly, but it looks like the claims are no longer specified, it's only the header. So you can amend this message (and the one in line 59 as well).